### ISSUE TITLE
rRNA not well annotated with Barrnap

### ISSUE THREAD

**Issue by ireneortega** [tseemann/prokka#485]
Hi Torsten,

I noticed Barrnap doesn't annotate well rRNA. My genomes have an intergenic spacer region of 150 bp in 23S rRNA and PROKKA annotates different CDS in that region. For that reason, no 23S rRNA is annotated in the genome, which is not correct. Could you please solve that?

Thanks!

**Comment by ireneortega** [comment]
I usually use RNAmmer instead of Barrnap. For a reference genome I tested, RNAmmer found the same number as are in the NCBI annotation for that genome.

**Comment by LRT-14** [comment]
Do you mean that RNAmmer works ok, but Barrnap doesn't?

**Comment by jacquefm** [comment]
Is RNAmmer still available outside of prokka?

**Comment by SouraTR** [comment]
Yes, RNAmmer is available, you can run it outside of PROKKA and integrate the results into the PROKKA gff
